draft-ietf-quic-recovery.md

```diff
@@ -932,6 +932,10 @@ max_ack_delay:
 loss_detection_timer:
```

Comment:
rename this to "timer", with description "multi-modal timer used for loss detection and for sending probes."